src/provider/systemdmanager/systemdunit.rs

```diff
@@ -554,6 +575,40 @@ mod test {
             [Install]
             WantedBy=multi-user.target"#}
     )]
+    #[case::with_container_on_session_bus_stoptimeout(
+    BusType::Session,
```

Comment:
The indentation is different to the other test cases.

Reply:
fixed